src/libraries/System.Speech/tests/SpeechRecognizerTests.cs

```diff
@@ -17,9 +17,38 @@
 
 namespace SampleSynthesisTests
 {
+    [ConditionalClass(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoNorServerCore))] // No SAPI on Nano or Server Core
+    [SkipOnMono("No SAPI on Mono")]
     public static class SpeechRecognizerTests
     {
-        [Fact]
+        private static bool RecognizerInstalledAndEnabled()
+        {
+            if (PlatformDetection.IsMonoRuntime ||
+                PlatformDetection.IsWindowsNanoServer ||
+                PlatformDetection.IsWindowsServerCore)
```

Comment:
Yes, probably - I noticed that after I merged the other PR. If it's OK, I'd like to leave it as it's harmless, to avoid having to rerun all the outer loop again. I'll clean it up in master.